jet_tracking/context.py

```diff
@@ -82,174 +166,171 @@ def __init__(self, signals):
-        self.percent_dropped = ds
-        self.signals.changeDroppedShots.emit(self.percent_dropped)
-
-    def update_peak_intensity(self, pi):
-        self.peak_intensity = pi
-        self.signals.changePeakIntensity.emit(self.peak_intensity)
+        self.image_calibration_steps = []
+        self.image_calibration_positions = []
+        self.contours = []
+        self.best_fit_line = []
+        self.best_fit_line_plus = []
+        self.best_fit_line_minus = []
+        self.com = []
+        self.find_com_bool = False
+        self.dilate = None
+        self.erode = None
+        self.open_operation = None
+        self.close_operation = None
+        self.brightness = None
+        self.contrast = None
+        self.blur = None
+        self.threshold = 110
 
-    def update_jet_radius(self, r):
-        self.radius = r
-        self.signals.changeJetRadius.emit(self.radius)
-
-    def update_jet_center(self, jc):
-        self.center = jc
-        self.signals.changeJetCenter.emit(self.center)
-
-    def update_max_intensity(self, mi):
-        self.max = mi
-        self.signals.changeMaxIntensity.emit(self.max)
+    def update_live_graphing(self, live):
+        """
+        Updates the live graphing mode.
 
-    def update_background(self, bgn):
-        self.bg = bgn
-        self.signals.changeBackground.emit(self.bg)
+        Parameters:
+            live (bool): A boolean value indicating whether live graphing is enabled or not.
 
-    def update_live_graphing(self, live):
+        """
         self.live_data = live
         self.signals.changeRunLive.emit(self.live_data)
+        self.signals.updateRunValues.emit(self.live_data)
 
     def update_calibration_source(self, cal_src):
+        """
+        Updates the calibration source.
+
+        Parameters:
+            cal_src (str): The calibration source to be used.
+
+        """
         self.calibration_source = cal_src
         self.signals.changeCalibrationSource.emit(self.calibration_source)
 
+    def update_num_cali(self, nc):
+        """
+        Updates the number of calibration points.
+
+        Parameters:
+            nc (int): The number of calibration points.
+
+        """
+        self.num_cali = nc
+        self.signals.changeNumberCalibration.emit(self.num_cali)
+
     def update_percent(self, p):
         """
         changes the percent threshold and emits a signal to the thread
         so that the range of allowed values and the graph will get updated
+
+        Parameters:
+            p (int): The percent threshold value.
+
         """
         self.percent = p
         if self.calibrated:
             self.signals.changePercent.emit(self.percent)
 
     def update_graph_averaging(self, avg):
         """
-        changes the number of points to average on the graph
+        updates the number of points to average on the graph.
+
+        Parameters:
+            avg (int): The number of points to average.
+
```

Comment:
Are you planning to generate sphinx html docs from this? There is some reformatting you can do if you want to plug into that system.

Reply:
Hmm, that might be a good idea in the future!